src/components/Tooltip/Tooltip.tsx

```diff
@@ -4,17 +4,15 @@ import React, { CSSProperties, isValidElement, PureComponent, ReactElement } fro
 import classnames from "classnames";
 import { Modifier } from "react-popper";
 import { isFunction } from "lodash-es";
-import Dialog from "../Dialog/Dialog";
+import Dialog, { DialogProps } from "../Dialog/Dialog";
 import { AnimationType, BASE_SIZES_WITH_NONE, HideShowEvent, JustifyType } from "../../constants";
 import { DialogPosition } from "../../constants/positions";
-import VibeComponentProps from "../../types/VibeComponentProps";
 import { TooltipArrowPosition, TooltipTheme } from "./TooltipConstants";
 import { ElementContent } from "../../types/ElementContent";
 import { MoveBy } from "../../types/MoveBy";
 import "./Tooltip.scss";
 
-// TODO TS-migration extend DialogProps, once Dialog is migrated to TS
-export interface TooltipProps extends VibeComponentProps {
+export interface TooltipProps extends Omit<DialogProps, "content"> {
```

Comment:
But why we ignore content prop?we support it both for dialog and tooltip?

Reply:
It's different types